### DISCUSSION TITLE
SSA 2024 in Anchorage, Alaska (April 29 to May 3)

### DISCUSSION THREAD

**Discussion by carltape** [SPECFEM/specfem3d#1663] [General]
Please consider attending SSA 2024 in Anchorage, Alaska (April 29 to May 3). Abstracts are due January 10th, and registration is expected to open a week or so before this.
https://meetings.seismosoc.org/
https://meetings.seismosoc.org/submit/
If you are able to attend, I encourage you to arrive Sunday April 28, in order to join the science seminar cruise into Prince William Sound on Monday. On Tuesday there is another field trip option, as well as workshops. And in the afternoon: likely a Specfem mini-workshop (with remote participation as an option) in the afternoon, as well as a dinner gathering. Details will follow soon.

There is one session of particular interest:

3D Wavefield Simulations: From Seismic Imaging to Ground Motion Modelling

Advances in numerical methods and continued evolution of computer hardware and high-performance computing infrastructures have made it now routine to simulate full 3D seismic wave propagation at local, regional and global scales. This capability and the growing efficiency in accomplishing it result in a broad range of applications, from ground motion simulation and scenario earthquakes incorporating 3D models, physics-based fault rupture dynamics simulations, to simulation-based seismic imaging methods such as full-waveform inversion. By taking advantage of ever-growing seismic observations, these applications have considerably expanded our understanding of seismic hazard, earthquake physics and regional and global tectonics. This session invites any seismic contributions that leverage the capabilities of full-wave simulations, including applications related to earthquake ground motion modelling, kinematic and dynamic rupture simulations, ambient noise and seismic imaging at all scales as well as other novel applications.

Conveners
Qinya Liu, University of Toronto
Ebru Bozdag, Colorado School of Mines
Bryant Chow, University of Alaska Fairbanks
Andreas Fichtner, ETH Zurich
Erin Wirth Moriarty, U.S. Geological Survey
Artie Rodgers, Lawrence Livermore National Laboratory

**Comment by ljhwang** [comment]
Also consider the following session:

Translating Seismic Imaging into Geodynamic Understanding 
Seismic imaging provides valuable information about the subsurface of the Earth. Travel times and waveforms from natural and controlled sources can be used to construct 2D and 3D velocity, attenuation and anisotropy models of the Earth’s interior from the core to the crust. These seismically determined 30 quantities are subsequently interpreted into physical properties and integrated into geodynamic models to explore a wide range of dynamical processes to understand Earth’s past, present and future evolution. This session seeks to increase the exchange between seismic and geodynamic model communities to better understand data and model uses and limitations. Contributions that explore applications of seismic imaging results in computational models that increase our understanding of Earth dynamics, model uncertainty and data resolution, and software tools used to construct model data are welcome. This includes but not limited to dynamics of the deep Earth including core-mantle interaction, mantle convection and mantle plumes; dynamics of the lithosphere including subduction zones, rifting and glacial isostatic adjustment; and dynamics of the crust including fault, geothermal and volcanic systems.

Conveners Ebru Bozdag, Colorado School of Mines
Rebecca Fildes, University of California, Davis 
Menno Fraters, University of Florida
Lorraine J. Hwang, University of California, Davis
Andrew Lloyd, Lamont-Doherty Earth Observatory, Columbia University
Brandon VanderBeek, Università di Padova
